lib/core.js

```diff
@@ -555,6 +555,8 @@ Archiver.prototype.append = function(source, data) {
  * @return {this}
  */
 Archiver.prototype.bulk = function(mappings) {
+  console.log('Archiver.bulk() deprecated since 0.21.0');
```

Comment:
Yes I know for the alias that's why I specified (or `console.error`). 

+1 about the fall back, you can just test `process && process.emitWarning !== undefined` (where `process` is in case it's beeing used in the navigator)